src/common/arrow/arrow_array_scan.cpp

```diff
@@ -1,5 +1,6 @@
 #include "common/arrow/arrow_converter.h"
 #include "common/exception/runtime.h"
+#include "common/types/int128_t.h"
```

Comment:
Why do we need to add the include in this PR?

Reply:
Because we remove certain header from arrow_converter.h